index.js

```diff
@@ -15,14 +15,36 @@ const {
 const quickInterval = QUICK_NAP_DURATION_SECONDS * 1000,
     slowInterval = SLOW_NAP_DURATION_MINUTES * 60_000
 
+const {
+    readSettings,
+    checkForFocusBearInstall,
+} = require('./src/applescript/fs.js')
+
+const {
+    setCalsToExclude,
+    setEventsToExclude,
+} = require('./src/applescript/calendar.js')
+
+const { setNagState } = require('./src/intention.js')
+
+async function setSettings() {
+    const { excluded_calendars, excluded_events } = await readSettings()
+    await setCalsToExclude(excluded_calendars)
+    await setEventsToExclude(excluded_events)
+    await setNagState()
+}
+
 async function main() {
     console.log('Late No More Online..')
+    await setSettings()
+
     if (IS_TESTING) await addTestEvents()
     else syncCalendarsToUpcoming()
     checkUpcomingForMeetings()
 
     setInterval(checkUpcomingForMeetings, quickInterval)
     setInterval(syncCalendarsToUpcoming, slowInterval)
+    setInterval(setNagState, 1000 * 60 * 60 * 24)
```

Comment:
can we have a constant for this? ONE_DAY_IN_SECONDS

Reply:
Done in latest commit.